build.gradle.kts

```
@@ -34,10 +32,10 @@ buildscript {
 
 plugins {
     alias(libs.plugins.baselineProfile) apply false
-    alias(libs.plugins.binaryCompatibility)
-    alias(libs.plugins.dokka)
     alias(libs.plugins.poko) apply false
+    alias(libs.plugins.binaryCompatibility)
     alias(libs.plugins.spotless)
+    id("org.jetbrains.dokka")
```

Can we move dokka back into the `[plugins]` section of `libs.versions.toml` so this can be tidier?
```suggestion
    alias(libs.plugins.dokka)
```

---

Unfortunately we can't due to a [gradle issue](https://github.com/gradle/gradle/issues/20084). Added a link to it!